source/extensions/filters/http/transformation/inja_transformer.h

```diff
@@ -56,6 +59,8 @@ class TransformerInstance {
   const nlohmann::json &context_;
   const std::unordered_map<std::string, std::string> &environ_;
   const envoy::config::core::v3::Metadata *cluster_metadata_;
+  std::unordered_map<std::string, std::string> pattern_replacements_;
```

Comment:
nit: in envoy `absl::flat_hash_map` is usually used instead of `std::unordered_map`

Reply:
nods, I saw we used `unordered_map` here and used the same for the consistency. I'll make the change.